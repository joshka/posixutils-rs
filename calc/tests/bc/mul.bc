a = 2 * 2
scale(a)
a

scale = 2
a = 2.000 * 2.000
scale(a)
a

scale = 10
a = 2.000 * 2.000
scale(a)
a

scale = 10
a = 2.000 * 2.0
scale(a)
a

scale = 10
a = 2.0 * 2.000
scale(a)
a

scale = 100
890901.1340 * 234.93124012384009

quit
