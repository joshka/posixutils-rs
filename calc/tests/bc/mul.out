0
4
3
4.000
6
4.000000
4
4.0000
4
4.0000
209300508.238355436615662060
